### DISCUSSION TITLE
Allow seek time in PIP

### DISCUSSION THREAD

**Discussion by cvphat** [ReVanced/revanced-discussions#226] [Question]
Yeah, like the title. 

I would like to have a feature that I can skip back or forward video by 5s or 10s in PIP mode

It is useful with me because I usually use PIP and read the news in the same time.

Thank you so much Revanced team.

**Comment by anil-al** [comment]
Double tab would interfere with resize action, buttons for 10 second forward or backward would work, though.

**Reply by cvphat** [reply]
Yes, Add back and forward button are a good idea

**Comment by CnC-Robert** [comment]
PiP is not handled by ReVanced but by android so you propably cant do something like this

**Reply by cvphat** [reply]
There is a different between PIP of Youtube and PIP of Browser. So I think it can be customized by app

**Reply by Amanse** [reply]
it can be customised, but only buttons can be added, it doesn't support double tapping
